src/snowflake/cli/plugins/nativeapp/teardown_processor.py

```diff
@@ -66,7 +66,7 @@ def drop_generic_object(
             cc.message(f"Dropped {object_type} {object_name} successfully.")
 
     def _application_objects_to_str(
-        self, application_objects: ApplicationOwnedObject
```

Comment:
not sure why mypy didn't flag this one when IntelliJ did /shrug

Reply:
I thought 3.8 does not allow `list[Type]` yet only `List[Type]` is accepted? But if it does, then great.

Reply:
we have `from __future__ import annotations` in all files, so annotations are only saved as string literals to be evaluated on-demand. Since the app never reads this particular annotation, it's not an issue (otherwise it would also crash on all the `X | Y` we use instead of `typing.Union`

Reply:
(i also tested this manually on 3.8 and it was fine)

Reply:
Aah yes, I forgot about the `from __future__ import annotations` change to code files. Makes sense.